### PR TITLE
Workflow

### PR DESCRIPTION
**Pull request by x566chen**
GitHub page deploy action

### CODE REVIEW

File: .github/workflows/Build&Deploy.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ jobs:
     - name: Build and Deploy
       uses: JamesIves/github-pages-deploy-action@releases/v3
       with:
```
Comment (eleven032) [inline .github/workflows/Build&Deploy.yml]: Action should be changed from push to merge

**Review by eleven032 — CHANGES_REQUESTED** [review]

**Comment by bee0060** [comment]
Does the 17 commits all valid for other developers? If not, please combine them.

File: build/asset-manifest.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "static/js/2.6541b97b.chunk.js.map": "./static/js/2.6541b97b.chunk.js.map",
+    "index.html": "./index.html",
+    "precache-manifest.72fcae8e2d0624506737ad8b48bb160a.js": "./precache-manifest.72fcae8e2d0624506737ad8b48bb160a.js",
+    "service-worker.js": "./service-worker.js",
+    "static/css/main.e9f78689.chunk.css.map": "./static/css/main.e9f78689.chunk.css.map",
+    "static/js/2.6541b97b.chunk.js.LICENSE": "./static/js/2.6541b97b.chunk.js.LICENSE"
+  },
+  "entrypoints": [
+    "static/js/runtime-main.f4662f19.js",
+    "static/js/2.6541b97b.chunk.js",
```
Comment (bee0060) [inline build/asset-manifest.json]: Are names of chunks dynamic? 

If you build again, will these files names all change? 

If yes, do you really need to keep this in git?

**Comment by bee0060** [comment]
emm... I saw many files were generated by build, do you really want to keep them in git? why?

**Pull request merged**